src/Extension/PhiremockProcess.php

```diff
@@ -89,52 +77,32 @@ private function setUpProcessCompatibility()
      */
     private function initProcess($ip, $port, $debug, $expectationsPath, $phiremockPath)
     {
-        $this->process = new Process(
-            $this->getCommandPrefix()
-            . "{$phiremockPath} -i {$ip} -p {$port}"
-            . ($debug ? ' -d' : '')
-            . ($expectationsPath ? " -e {$expectationsPath}" : '')
-        );
-    }
-
-    /**
-     * @param string $ip
-     * @param int    $port
-     * @param bool   $debug
-     * @param string $expectationsPath
-     * @param string $phiremockPath
-     */
-    private function logPhiremockCommand($ip, $port, $debug, $expectationsPath, $phiremockPath)
-    {
+        $commandline = [
+            $phiremockPath,
+            '-i',
+            $ip,
+            '-p',
+            $port
+        ];
         if ($debug) {
-            echo 'Running ' . $this->getCommandPrefix()
-                . "{$phiremockPath} -i {$ip} -p {$port}"
-                . ($debug ? ' -d' : '')
-                . ($expectationsPath ? " -e {$expectationsPath}" : '') . PHP_EOL;
+            $commandline[] = '-d';
+        }
+        if ($expectationsPath) {
+            $commandline[] = '-e';
+            $commandline[] = $expectationsPath;
         }
-    }
-
-    /**
-     * @return string
-     */
-    private function getCommandPrefix()
```

Comment:
`symfony/process` adds `exec` to commandline in UNIX systems if commandline defiened as array:
```php
        if (is_array($commandline = $this->commandline)) {
            $commandline = implode(' ', array_map(array($this, 'escapeArgument'), $commandline));

            if ('\\' !== DIRECTORY_SEPARATOR) {
                // exec is mandatory to deal with sending a signal to the process
                $commandline = 'exec '.$commandline;
            }
        }
```